### PR TITLE
microverse trial project code review

### PR DESCRIPTION
**Pull request by Zia123456**
This pull request was created to review the microverse trial project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# code_review_project_1
```
Comment (Zia123456) [inline README.md:1]: # Very Bad Readme file
I suggest you refer to the below Readme template.
[This Template](https://github.com/microverseinc/readme-template).

File: index.babel [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    from red to green.
+  - As you start to build out your 
+    project, when tests are failing, 
+    you should get helpful errors 
+    along the way!
+    ************/
+
+// PLEASE NOTE: Adding global style rules using the * selector, or by adding rules to body {..} or html {..}, or to all elements within body or html, i.e. h1 {..}, has the potential to pollute the test suite's CSS. Try adding: * { color: red }, for a quick example!
+
+// Once you have read the above messages, you can delete all comments.
```
Comment (Zia123456) [inline index.babel:27]: # Unnecessary file
I suggest you remove unnecessary files from your project directory.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
```
Comment (Zia123456) [inline index.html]: # Bad practices of HTML & CSS
### Blank line
Leaving unnecessary blank line make your code messy. I suggest you remove unnecessary blank lines.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+          <nav id="nav-bar">
+            <ul>
+              <li><a href="#div1" class="nav-link">Features</a></li>
+              <li><a href="#div2" class="nav-link">How it will work</a></li>
+              <li><a href="#foot" class="nav-link">About us</a></li>
+            </ul>
+          </nav>
+        </div>
+        
+      </div>
```
Comment (Zia123456) [inline index.html]: # Bad practice of HTML & CSS
### Blank line
As mentioned above.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+    </header>
+    <br>
+    <div class="cont">
+    <div id="div1">
```
Comment (Zia123456) [inline index.html]: I suggest you use semantic HTML tags.
Instead of div you can use header, section, nav tags.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+        </div>
+      </div>
+      
+
+
+
+      <div class="prices">
+        <div class="prices-head">
+        <h4 class="prices-headings">Laptop</h4>  
+        </div>
```
Comment (Zia123456) [inline index.html]: # Consider indentation
I suggest you put indentation for children's tags.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <header id="header">
+      <div id="main-header">
+        <div id="logo">
+          <img
+            src="https://upload.wikimedia.org/wikipedia/commons/5/53/Wikimedia-logo.png"
+            alt="the logo is not available"
+            id="header-img"
+            height="40"
+            width="40"
```
Comment (Zia123456) [inline index.html]: # Inline style
I suggest you do not use inline style, it makes your code missy and unreadable.

File: .github/workflows/linters.yml [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
+  nodechecker:
+    name: node_modules checker
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Check node_modules existence
+        run: |
+          if [ -d "node_modules/" ]; then echo -e "\e[1;31mThe node_modules/ folder was pushed to the repo. Please remove it from the GitHub repository and try again."; echo -e "\e[1;32mYou can set up a .gitignore file with this folder included on it to prevent this from happening in the future." && exit 1; fi
```
Comment (Zia123456) [inline .github/workflows/linters.yml:56]: # Linters Error
Before committing please make sure there are not linters error.

**Review by Zia123456 — COMMENTED** [review]
# Very bad code
There is not any bullet point in your code.

File: README.md [hunk 9]
```diff
@@ -1 +1,33 @@
...
```
Comment (Zia123456) [inline README.md:1]: # Good Readme file

File: index.html [hunk 10]
```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
```
Comment (Zia123456) [inline index.html:1]: # Clean and tide code
Your code looks clean and you used semantic tags and, completed the changes requirement.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @Zia123456    ,

### Status: Approved ✔️ ✔️ ✔️

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

Cheers and Happy coding!👏👏👏
## Optional Suggestion

- [x] Kindly make sure all of the suggestion you give includes an explanation, a "why", for each requested change, some the requested suggestion lack an explanation.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**